helpers/playerHelper.js

```diff
@@ -78,6 +87,7 @@ exports.parseCollection = function(players, subresources) {
       switch (resource) {
         case 'stats':
           player.stats = self.mapStats(p[idx + 1].player_stats);
+          player.points = self.mapPoints(p[idx + 1].player_points);
```

Comment:
This project is great - thank you - no need to apologize for not having free time :)

Personally, I might have some free time between jobs in a little bit, so I _might_ be able to help with tests if you think they'd be valuable.